inspector_go/rpc/req.go

```diff
@@ -0,0 +1,144 @@
+package rpc
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/umee-network/price_inspector/config"
+	otypes "github.com/umee-network/umee/v6/x/oracle/types"
+)
+
+func GetAcceptedDenoms(codec testutil.TestEncodingConfig, api string) []string {
```

Comment:
please use `/client` (in the umee repo). Or update it if necessary.

Reply:
Updated